npm-debug.log

```diff
@@ -0,0 +1,19 @@
+0 info it worked if it ends with ok
```

Comment:
Kill it with fire!

(Or add this to your global or local .gitignore, remove the git cache, and recommit and push up again)